pages/index.html

```diff
@@ -7,65 +7,52 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>&hearts;home&hearts;</title>
 </head>
-
-
-<body>
+  
+  <body>
     <header>
       <h1>&hearts;homepage&hearts;</h1>
       <nav>
-      <div class="dropdown">
-          <span>navigation:</span>
-          <div class="dropdown-content">
-              <a href="index.html" >home</a>
-              <br><a href="about.html">about</a>
-              <br><a href="portfolio.html">portfolio</a>
-            </div>
-        </div>
-      </nav>
-      <br>
-    </header>
+      <ul>
+        <li><a href="index.html" >home</a></li>
+        <li><a href="about.html">about</a></li>
+        <li><a href="portfolio.html">portfolio</a></li>
+      </ul>
+    </nav>
+  </header>
```

Comment:
The indentation looks a little off here. I would expect it to look something like:
```
    <body>
        <header>
            <h1>&hearts;homepage&hearts;</h1>
            <nav>
                <ul>
                    <li><a href="index.html" >home</a></li>
                    <li><a href="about.html">about</a></li>
                    <li><a href="portfolio.html">portfolio</a></li>
                </ul>
            </nav>
        </header>
```